libs/showToast.js

```diff
@@ -0,0 +1,4 @@
+async function showToast(message) {
```

Comment:
oh, showToast is synchronous function. So no need `async` keyword

Reply:
i remove async